src/transforms/select.jl

```diff
@@ -55,19 +97,14 @@ function apply(transform::Select, table)
   # original columns
   cols = Tables.columns(table)
 
-  # selected columns
-  scols = [Tables.getcolumn(cols, name) for name in select]
-
   # rejected columns
   rcols = [Tables.getcolumn(cols, name) for name in reject]
 
-  # table with selected columns
-  𝒯 = (; zip(select, scols)...)
-  stable = 𝒯 |> Tables.materializer(table)
-
-  stable, (reject, rcols, sperm, rinds)
+  TableSelection(table, select), (reject, rcols, sperm, rinds)
 end
 
+revert(::Select, newtable::TableSelection, cache) = newtable.table
+
```

Comment:
Move this method to after the general revert method and add a one-line comment explaining that this is for method for performance optimization only.

Reply:
Done!
Resolved in 6199b20.